### PR TITLE
Add r-mapdata and r-ggfortify

### PR DESCRIPTION
**Pull request by ocefpaf**
See https://cran.r-project.org/web/packages/mapdata/index.html and https://cran.r-project.org/web/packages/ggfortify/index.html

### CODE REVIEW

File: recipes/r-mapdata/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-maps >=2.0_7
+    - posix  # [win]
+    - m2w64-toolchain  # [win64]
```
Comment (isuruf) [inline recipes/r-mapdata/meta.yaml]: Add `toolchain` for unix

Comment (ocefpaf) [inline recipes/r-mapdata/meta.yaml]: The Windows `m2w64toolchain` is not the same thing. Note that all the r packages so far have this: https://github.com/conda-forge/r-maps-feedstock/blob/master/recipe/meta.yaml#L31 (only Windows), and then uses `gcc` on both `OS X` and `Linux`.

Comment (isuruf) [inline recipes/r-mapdata/meta.yaml]: I don't think it's using `gcc` on `OS X`. See the travis build

Comment (ocefpaf) [inline recipes/r-mapdata/meta.yaml]: Yep.

**Pull request merged**